packages/cascara/src/modules/DataDate/index.js

```diff
@@ -0,0 +1 @@
+export { default, propTypes } from './DataDate';
```

Comment:
Are we actually using the prop types for this module outside of the module itself?

Reply:
propTypes is not being used outside of this module. Code removed.